Src/library/api_wrapper/api/static_module.e

```diff
@@ -0,0 +1,46 @@
+note
+	description: "Objec representing {STATIC_MODULE}."
```

Comment:
typo   `Objec` ... also I am sure we could have a more useful comment.

Reply:
fixed